tools/devtool

```diff
@@ -285,9 +285,17 @@ run_devctr() {
     [[ -t 0 ]] && docker_args+=("-i")
     [[ -t 1 ]] && docker_args+=("-t")
 
+    # Try to pass these environments from host into container for network proxies
+    proxies=(http_proxy HTTP_PROXY https_proxy HTTPS_PROXY no_proxy NO_PROXY)
+    for i in "${proxies[@]}"; do
+        if [[ ! -z ${!i} ]]; then
+            docker_args+=("--env $i=${!i}")
```

Comment:
This pushes two args (`--env` and `$i=${!i}`) as a single item in the array. That's why you got that error last time.

You need to push each arg as its own array item. E.g:
```bash
docker_args+=("--env") && docker_args+=("$i=${!i}")
```

Reply:
Ok it works, thank you for your tips